pkg/controller/controller_service.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2018 The Kubeflow Authors
```

Comment:
I think the file name has redundancy, could we name it  pkg/controller/service.go

Reply:
ditto

Reply:
Actually, this file is split from pkg/controller/controller.go.